site/src/App/guides/testing-guide.tsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+
+import { Page } from '../../types';
+import { TextStack } from '../TextStack/TextStack';
+import {
+  Heading,
+  Divider,
+  Text,
+  BulletList,
+  Bullet,
+  Alert,
+} from '../../../../lib/components';
+import { Link, ExternalLink } from '../Documentation/Link';
+import Code from '../Code/Code';
+
+const page: Page = {
+  title: 'Testing guide',
+  Component: () => (
+    <TextStack>
+      <Heading level="2">Testing guide</Heading>
+
+      <Text tone="secondary">
+        This document aims to provide guidance and best practices for testing
+        applications that use Braid.
+      </Text>
+
+      <Alert tone="info">
+        The following info is opinions we&apos;ve formed from our experience
+        building Braid and other React applications. Each team should decide
+        what testing approach works best for them.
+      </Alert>
+
+      <Divider />
+
+      <Heading level="3">How do I render Braid components in jest?</Heading>
+
+      <Text>
+        Braid doesn&apos;t require any special setup to render within{' '}
+        <ExternalLink href="https://jestjs.io/">jest</ExternalLink>, however,
+        all Braid components must be wrapped in a provider. Unit tests should
+        use the{' '}
+        <Link to="/components/BraidTestProvider">BraidTestProvider</Link>. This
+        component performs the same role as the{' '}
+        <Link to="/components/BraidProvider">BraidProvider</Link> but smooths
+        over a few issues that are unique to unit testing. It also sets a
+        default theme for you as themes are largely irrelevant when unit
+        testing.
+      </Text>
+
+      <Heading level="3">Should I use Enyzme or React Testing Library?</Heading>
+
+      <Text>
+        While{' '}
+        <ExternalLink href="https://enzymejs.github.io/enzyme/">
+          Enzyme
+        </ExternalLink>{' '}
+        will work perfectly fine, we recommend{' '}
+        <ExternalLink href="https://testing-library.com/docs/react-testing-library/intro">
+          React Testing Library
+        </ExternalLink>{' '}
+        and use it internally in the Braid codebase. We feel it encourages you
+        to write good tests.
+      </Text>
+
+      <Text>
+        <ExternalLink href="https://testing-library.com/docs/react-testing-library/intro">
+          React Testing Library
+        </ExternalLink>{' '}
+        has the following great features:
+      </Text>
+
+      <BulletList>
+        <Bullet>Forces you to treat your components like DOM elements</Bullet>
+        <Bullet>
+          <ExternalLink href="https://testing-library.com/docs/dom-testing-library/api-queries#byrole">
+            Role based querying
+          </ExternalLink>
+        </Bullet>
+        <Bullet>No shallow rendering</Bullet>
+      </BulletList>
+
+      <Heading level="3">What is a good unit test?</Heading>
```

Comment:
What do you think about "How we write unit tests" as a heading for this section?